src/bundle/Resources/config/routing.yml

```diff
@@ -570,6 +570,13 @@ ezplatform.content.translate:
         _controller: 'EzPlatformAdminUiBundle:ContentEdit:translate'
         fromLanguageCode: ~
 
+ibexa.content.translate_with_location:
+    path: /content/{contentId}/location/{locationId}/translate/{toLanguageCode}/{fromLanguageCode}
+    methods: ['GET', 'POST']
+    defaults:
+        _controller: '\EzSystems\EzPlatformAdminUiBundle\Controller\ContentEditController::translateAction'
```

Comment:
Close... :sweat_smile: 

```suggestion
        _controller: 'EzSystems\EzPlatformAdminUiBundle\Controller\ContentEditController::translateAction'
```

Also, it could be changes to use `controller` route property, see YAML declarations in the Symfony documentation: https://symfony.com/doc/current/routing.html#creating-routes-in-yaml-xml-or-php-files

```yaml
# config/routes.yaml
blog_list:
    path: /blog
    # the controller value has the format 'controller_class::method_name'
    controller: App\Controller\BlogController::list

    # if the action is implemented as the __invoke() method of the
    # controller class, you can skip the '::method_name' part:
    # controller: App\Controller\BlogController
```

Reply:
We have such naming in other parts of the system: https://github.com/ezsystems/date-based-publisher/blob/master/bundle/Resources/config/routing.yaml

Reply:
And it works because that's basically where Symfony puts this info - in `Request` attributes. So I don't really mind that it's there, just keep in mind that it's not how you're supposed to do it "officially", according to docs.